assets/scss/components/_button.scss

```diff
@@ -322,7 +304,7 @@ Class                   | Description
 	@include color-all($C_textSecondary, true);
 	@include standardBorder(all);
 	outline: none;
-	background: transparentize($C_mediumGray, .88) !important;
+	background: transparentize($C_mediumGray, .88);
```

Comment:
If this value is set to something lighter, or the button appears on a lighter background, it still shows.